src/libraries/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -248,6 +248,8 @@ namespace System.Drawing
 
         public static Color Purple => new Color(KnownColor.Purple);
 
+        public static Color RebeccaPurple => new Color(KnownColor.RebeccaPurple);
+
```

Comment:
Can you also update the tests, e.g. https://github.com/dotnet/runtime/blob/9162830bc651fbff0ab1957c0155d4ecf375109e/src/libraries/System.Drawing.Primitives/tests/ColorTests.cs#L16 ?

Reply:
ok i will do this soon.